bot/cogs/games/games.py

```diff
@@ -39,3 +39,19 @@ async def ball8(self, ctx: Context, *, question: str) -> None:
         embed.add_field(name="Answer", value=answer)
         
         await ctx.send(embed=embed)
+    @command()
+    async def flip_coin():
+        def coin_filp():
```

Comment:
We require all functions to have proper docstrings as these are what will be displayed in the `>>help` even tho this command is pretty self-descriptive, it should still have a docstring, same goes fro `dice` command

Reply:
Actually, @ItsDrike we shouldn't worry about the `>>help` command since it will be replaced by the paginator once it is released.

Reply:
@e-exe even with it replaced docstrings will still be used as a command descriptions even with it being paginated